modules/oe_theme_paragraphs_contact/oe_theme_paragraphs_contact.info.yml

```
@@ -0,0 +1,7 @@
+name: OpenEuropa Theme Paragraphs Contact
+type: module
+description: Companion module for the OE Paragraphs Contact module
+package: OpenEuropa
+core: 8.x
+dependencies:
+  - oe_paragraphs:oe_paragraphs_contact
```

```suggestion
  - oe_paragraphs:oe_paragraphs_contact
  - oe_theme:oe_theme_content_entity_contact
```
If we want to use Full view mode of Contact entity.